Gemfile

```diff
@@ -5,6 +5,5 @@ gemspec
 group :development, :test do
   gem 'rake', '~> 13.0.1'
   gem 'pry-byebug'
-  gem 'rubocop'
-  gem 'factory_bot'
```

Comment:
is factorybot a dependency of another gem? I see it's removed here but still in the lock file

Reply:
It gets added via the `gemspec` file (interestingly the template does not manage the Gemfile directly)